storybook/src/components/Colour.tsx

```diff
@@ -0,0 +1,45 @@
+import clsx from "clsx";
+import { twMerge } from "tailwind-merge";
+
+type ColourProps = {
+  className: string;
+  rgb: string;
+  textDark?: boolean;
+  description?: string;
+};
+
+export const Colour = ({
```

Comment:
Haha, I think this is an important decision we need to make. Because all the Extension texts are written in British-English, but the libraries, like Tailwind, are in American-English. I'll update the code in order to reflect your comment, @mateuszjasiuk !